mail/getmail/Portfile

```diff
@@ -1,28 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 PortSystem 1.0
 PortGroup python 1.0
 
 name                getmail
-version             5.6
+version             5.13
 categories          mail python
 platforms           darwin
 maintainers         nomaintainer
 license             GPL-2
 description         extensible mail retrieval system with POP3, IMAP4, SSL support
-long_description    getmail version 4 is a flexible, extensible mail retrieval \
-                    system with support for POP3, IMAP4, SSL variants of both, \
-                    maildirs, mboxrd files, external MDAs, arbitrary message \
-                    filtering, single-user and domain-mailboxes, and many \
-                    other useful features.
+long_description    getmail version 5 is a flexible, extensible mail retrieval \
+					system with support for POP3, IMAP4, SSL variants of both, \
+					maildirs, mboxrd files, external MDAs, arbitrary message \
+					filtering, single-user and domain-mailboxes, and many \
+					other useful features.
 # getmail does not contain binary code
 supported_archs     noarch
 
 homepage            http://pyropus.ca/software/getmail/
 master_sites        ${homepage}old-versions/
-checksums           rmd160  26a36c404cc07c29704aedc862cb8af0a376af12 \
-                    sha256  460d2c8834936df88d594095d789c4585edca9b0bdbeded9f6267f0d90dcd59a
+checksums           rmd160  c2f43c7d3f6a87e9663cb6b7d2d3ed77b66123a1 \
+                    sha256  04d52f6475f09e9f99b4e3d2f1d2eb967a68b67f09af2a6a5151857f060b0a9d \
+					size 199073
 dist_subdir         ${name}
 patchfiles          patch-setup.py.diff
 
+python.versions     26 27
```

Comment:
Adding python26 makes no sense here, so please just remove this line.

Reply:
And indeed, given that Python 2 becomes unsupported in five months, we should only be adding Python 3 at this point.

Reply:
OK, but get mail doesn't support Python 3 yet. So we are stuck with python27. There is discussion on the get mail mailing list about moving to Python 3, but it doesn't have a large priority. Although Python 2 supports stops next year, that doesn't mean it stops working.